toqito/matrix_ops/tests/test_vector_to_density_matrix.py

```diff
@@ -0,0 +1,56 @@
+"""Test vector_to_density_matrix."""
+import numpy as np
+import pytest
+
+from toqito.matrix_ops import vector_to_density_matrix
+
+
+def test_vector_to_density_matrix_with_1d_vector():
+    """Test conversion of 1D vector to density matrix."""
+    vector = np.array([1, 2, 3])
+    expected_density_matrix = np.array([[1, 2, 3],
+                                        [2, 4, 6],
+                                        [3, 6, 9]])
+    computed_density_matrix = vector_to_density_matrix(vector)
+    np.testing.assert_array_equal(computed_density_matrix, expected_density_matrix)
```

Comment:
There were issues with failures related to using ` np.testing.assert_array_equal` for `macOS`. It might be better to not use this in this PR as well. See the discussion in #423. 

We didn't get the chance to have this discussion yet but I think removing everything related to [`np.testing`](https://numpy.org/doc/stable/reference/routines.testing.html#test-support-numpy-testing) from all tests in `toqito` would be a better option. We could create a separate issue for the other modules. I remember @Bchass was interested in working on making these changes.

Reply:
Yep, I'll go ahead and remove this to just use `assert` statements. Good point, changed.